environment.js

```diff
@@ -0,0 +1,26 @@
+"use strict"
+
+function checkEnvironmentForConfig(config:Object) : Object {
```

Comment:
Thanks Christopher, I'll address your comments asap.

Ian

On 22 Aug 2016 10:59 pm, "Christopher Chedeau" <notifications@github.com>
wrote:
>
> In environment.js:
>
> > @@ -0,0 +1,26 @@ > +"use strict" > + > +function
checkEnvironmentForConfig(config:Object) : Object {
>
> you don't have @flow on the file, so : Object doesn't do anything fyi
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.